functests/4_latency/latency.go

```diff
@@ -48,6 +48,7 @@ var (
 	latencyTestRuntime = defaultTestRuntime
 	maximumLatency     = defaultMaxLatency
 	latencyTestCpus    = defaultTestCpus
+	logName            = ""
```

Comment:
you can use it as local variable and pass it to all relevant methods, it less error-prone than re-assign the global variable each time

Reply:
So I am expecting you will have
```
Describe(..., func() {
...
var logName string
...
BeforeEach(func() {
logName = time.Now().Format("20060102150405")
pod := getPod(..., logName)
})

AfterEach(func() {
removeLog(node, logName)
})
})
```

Reply:
Done , thanks !